packages/shared/src/enterprise/models/licenseModel.ts

```diff
@@ -25,7 +25,6 @@ const licenseSchema = new mongoose.Schema({
   },
   orbSubscription: {},
   stripeSubscription: {},
-  billingPlatform: String,
```

Comment:
@jdorn Wanted to call this change out to you specifically to make sure this wasn't going to introduce any unwanted side effects.

I've removed this and upgrade the `getSubscriptionFromLicense` function so that we infer the `billingPlatform` based on whether the license has a `stripeSubscription` or `orbSubscription`. I've retained the `billingPlatform` property on the `SubscriptionInfo` type, though, as it is handy on the frontend.